website/docs/components/flyout/partials/code/component-api.md

```diff
@@ -18,6 +18,9 @@
   <C.Property @name="size" @type="enum" @values={{array "medium" "large" }} @default="medium">
     Sets the width of the Flyout.
   </C.Property>
+  <C.Property @name="returnFocusTo" @type="string">
```

Comment:
Let me know if you think I should add also an example in the "how to use" section.

Reply:
I think it's a good idea, but not a blocker

Reply:
Done. Can you check the updates? thanks